recipes/r-pubmed.miner/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = '1.0.16' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pubmed.miner
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pubmed.mineR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pubmed.mineR/pubmed.mineR_{{ version }}.tar.gz
+  sha256: 30da150e8baa9dc1b88cd83fb985a25c17cfcf680694cff95fd7d343868f9bb5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-r2html
+    - r-rcurl
+    - r-xml
+    - r-boot
+  run:
+    - r-base
+    - r-r2html
+    - r-rcurl
+    - r-xml
+    - r-boot
+
+test:
+  commands:
+    - $R -e "library('pubmed.mineR')"           # [not win]
+    - "\"%R%\" -e \"library('pubmed.mineR')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=pubmed.mineR
+  license: GPL-3
```

```suggestion
  license: GPL-3.0-only
```

According to:
* https://cran.r-project.org/web/packages/pubmed.mineR/index.html
* https://spdx.org/licenses/GPL-3.0-only.html